service-deployment/deploy.py

```diff
@@ -223,6 +265,28 @@ def bootstrap_service(service_config):
 
 
 
+def copy_arrangement_service(serv, service_config):
+
+    service_list = service_config['servicelist']
+
+    if 'copy' not in service_list[serv]:
+        return
+
+    for target in service_list[serv]['copy']:
+        dst = "bootstrap/{0}/{1}".format(serv, target['dst'])
+        src = target['src']
+
+        if os.path.exists(dst) == False:
+            shell_cmd = "mkdir -p {0}".format(dst)
+            error_msg = "failed to mkdir -p {0}".format(dst)
+            execute_shell(shell_cmd, error_msg)
+
+        shell_cmd = "cp -r {0} {1}".format(src, dst)
+        error_msg = "failed to copy {0}".format(src)
+        execute_shell(shell_cmd, error_msg)
+
+
+
 def copy_arrangement(service_config):
```

Comment:
>copy_arrangement [](start = 4, length = 16)

maybe to reuse copy_arrangement_service here.

Reply:
Yes, I will change it.